specification/reservations/resource-manager/readme.md

```diff
@@ -24,6 +24,20 @@ To see additional help and options, run:
 
 These are the global settings for the Reservations API.
 
+``` yaml
+openapi-type: arm
+tag: package-2021-07-01
+```
```

Comment:
We need to remove the same section at line 41-44, and the issues in go and python SDK generation should be gone.
We can only have one of these sections that specifies the default tag in `readme.md` for autorest

Reply:
Thanks so much!

Reply:
The go one is still failing with: FATAL: System.ArgumentException: An item with the same key has already been added. Key: Client

Reply:
Strange.  By process of elimination, I figured out that it was the operation name of the new API.  I changed it and got it to pass locally.